pkl-core/src/main/java/org/pkl/core/util/yaml/snake/YamlResolver.java

```diff
@@ -48,15 +48,7 @@ public Tag resolve(String value, Boolean implicit) {
     return Tag.STR;
   }
 
-  private static final class ResolverTuple {
-    private final Tag tag;
-    private final Pattern regexp;
-
-    ResolverTuple(Tag tag, Pattern regexp) {
-      this.tag = tag;
-      this.regexp = regexp;
-    }
-
+  private record ResolverTuple(Tag tag, Pattern regexp) {
     @Override
     public String toString() {
       return "Tuple tag=" + tag + " regexp=" + regexp;
```

Comment:
Some inconsistency here with the `toString()` format. Might be worth deleting this and using the one that comes with `record` ?

Reply:
The `org.pkl.core.util.yaml.snake` package contains source code that is vendored from the SnakeYAML library.